resources/config.test.ci.edn

```diff
@@ -29,34 +29,34 @@
-                                                          :default-key-serde    "org.apache.kafka.common.serialization.Serdes$StringSerde"
-                                                          :default-value-serde  "org.apache.kafka.common.serialization.Serdes$StringSerde"
-                                                          :key-deserializer-encoding    "UTF8"
-                                                          :value-deserializer-encoding  "UTF8"
-                                                          :deserializer-encoding        "UTF8"
-                                                          :upgrade-from         "1.1"
-                                                          :channels             {:channel-1 {:worker-count [10 :int]
-                                                                                             :retry        {:count   [5 :int]
-                                                                                                            :enabled [true :bool]}}}}}
-            :tracer               {:enabled               [true :bool]
-                                   :custom-provider       ""}}}
+            :stream-router        {:default            {:application-id       "test"
+                                                        :bootstrap-servers    "localhost:9092"
+                                                        :stream-threads-count [1 :int]
+                                                        :origin-topic         "topic"
+                                                        :upgrade-from         "1.1"
+                                                        :channels             {:channel-1 {:worker-count [10 :int]
+                                                                                           :retry        {:type    [:linear :keyword]
+                                                                                                          :count   [5 :int]
+                                                                                                          :enabled [true :bool]}}}
+                                                        :producer             {:bootstrap-servers                     "localhost:9092"
+                                                                               :acks                                  "all"
+                                                                               :retries                               5
+                                                                               :max-in-flight-requests-per-connection 5
+                                                                               :enable-idempotence                    false
+                                                                               :value-serializer-class                "org.apache.kafka.common.serialization.StringSerializer"
```

Comment:
Because that's what this [config field] https://kafka.apache.org/23/javadoc/org/apache/kafka/clients/producer/ProducerConfig.html#VALUE_SERIALIZER_CLASS_CONFIG) is called. Since this approach generates config field names based on the provided configs, it's important to be consistent.